providers/ldap/ldap_client.go

```diff
@@ -95,7 +95,7 @@ func (l *LClient) newConn() (*ldap.Conn, error) {
 		certFound := false
 		for _, certFile := range certFiles {
```

Comment:
Dont loop through all the possibilities. We know where it is going to be, we control that, only do that one path